test/integration/api/audits.js

```diff
@@ -55,7 +55,6 @@ describe('/audits', () => {
               audits[0].details.should.eql({ data: {
                 actorId: david.actor.id,
                 email: 'david@getodk.org',
-                mfaSecret: null,
```

Comment:
I guess this means that `mfaSecret` will continue to exist in old audit log entries. I think that's fine though: it doesn't seem useful to scrub it from the audit log.